### ISSUE TITLE
Linter expects invalid syntax for multiline generics

### ISSUE THREAD

**Issue by spiffytech** [palantir/tslint#1830]
### Bug Report

- __TSLint version__: 4.0.3
- __TypeScript version__: 2.0.10
- __Running TSLint via__: any

#### TypeScript code being linted

```ts
// Makes more sense with long signatures that exceed the line length limit.
function fn<
    A extends Object,
    B extends Object  // tslint error here
>(a: A, b: B) {
    return [a, b];
}

```

with `tslint.json` configuration:

```json
{
    "extends": "tslint:recommended",
    "rules": {
        "variable-name": [
            true,
            "ban-keywords",
            "allow-pascal-case"
        ],
        "object-literal-sort-keys": false,
        "only-arrow-functions": false,
        "interface-name": [true, "never-prefix"]
    }
}
```

#### Actual behavior

tslint demands a trailing comma on the last item of the multiline generic declaration. A trailing comma there would be syntactically invalid (TS1009).

#### Expected behavior

Trailing comma should not be expected.

**Comment by nchen63** [comment]
duplicate of https://github.com/palantir/tslint/pull/1775
